src/lib/i18n.ts

```diff
@@ -27,19 +27,10 @@ export function setLocale(locale?: SiteLocale) {
 
 /**
  * Returns locale name for a given code in its own language.
- * This method only supports a few languages, to keep the bundle size small when used client-side.
- * To add more, see: https://github.com/adlawson/nodejs-langs/blob/master/data.js (not available in ESM)
+ * @example getLocaleName('nl') => 'Nederlands'
  */
 export function getLocaleName(code: string) {
-  const localeNamesByCode = {
-    'de': 'Deutsch',
-    'en': 'English',
-    'es': 'Español',
-    'fr': 'Français',
-    'it': 'Italiano',
-    'nl': 'Nederlands',
-  };
-  const localeName = localeNamesByCode[code as keyof typeof localeNamesByCode];
+  const localeName = new Intl.DisplayNames([code], { type: 'language' }).of(code);
```

Comment:
Not really part of this PR. But implementing the `FileLink` I discovered the `Intl.DisplayNames` API, which is a much more robust and future-proof implementation of this helper function.